### PR TITLE
working restaurant search based on user location and pick of color wh…

### PR DESCRIPTION
**Pull request by jkrejcik**
Seed file no longer fill db with restaurants. New restaurant is created based on each roll. The roll takes user current location (have to click allow location) and answer to questions (so far only colors which represent different cuisines). Next will be Favourite transport question which will increase radius from which random restaurant Is picked. Base is 500m.

I commented out lot of price method which are not needed and deleted extra questions forms but didn`t touch any JS existing files (only added new one which supplies user location).

### CODE REVIEW

File: app/controllers/restaurant_results_controller.rb [hunk 1]
```diff
@@ -37,50 +76,50 @@ def update
...
+    # cuisine = @colour_cuisine_key[params[:colour].to_sym] if params[:colour]
+    # cuisine ? Restaurant.where(cuisine: cuisine) : []
   end
 
   def colour
     @colour_cuisine_key = {
-      orange: "Chinese", brown: "Mexican", pink: "Jamaican", blue: "Drama", green: "Italian",
-      red: "Taiwanese", yellow: "Spanish", purple: "French", violet: "German", emerald: "Eritrean",
-      grey: "Russian", jasmine: "Japanese", myrtle: "Greek", navy: "Vietnamese", maroon: "Polish", cream: "American"
+      orange: "French", brown: "Chinese", pink: "Chinese", blue: "Italian", green: "Greek",
```
Comment (ronanoboyle) [inline app/controllers/restaurant_results_controller.rb]: Chinese is in twice, I can update this no worries, unless this was done purposefully?

**Review by ronanoboyle — APPROVED** [review]
Just the small change of Chinese being in twice, but I can do that?

File: app/controllers/restaurant_results_controller.rb (hunk 1 above)
Comment (jkrejcik) [reply app/controllers/restaurant_results_controller.rb]:
```suggestion
      orange: "French", brown: "Chinese", pink: "Vegetarian", blue: "Italian", green: "Greek",
```

**Comment by jkrejcik** [comment]
@ronanoboyle - changed one Chinese it to Vegetarian :-)

**Pull request merged**